### ISSUE TITLE
Difference in translations

### ISSUE THREAD

**Issue by Kukurijek** [SAP/cloud-sdk-ios-fiori#304]
**Describe the bug**

I took a look at the translations and noticed some differences:

In [German](https://github.com/SAP/cloud-sdk-ios-fiori/blob/main/Sources/FioriSwiftUICore/_localization/de.lproj/FioriSwiftUICore.strings
) translations the infinitive is used exclusively:
e.g. `"Tap to Sign" = "Zum Signieren tippen";`
But in [Serbian](https://github.com/SAP/cloud-sdk-ios-fiori/blob/main/Sources/FioriSwiftUICore/_localization/sr-latn-RS.lproj/FioriSwiftUICore.strings) and [Croatian](https://github.com/SAP/cloud-sdk-ios-fiori/blob/main/Sources/FioriSwiftUICore/_localization/hr.lproj/FioriSwiftUICore.strings) translations is used imperative instead of infinitive and even worse there is a mix in way of addressing someone.
e.g. `"Tap to Sign" = "Dodirnite da biste potpisali";` and `"Tap to Sign" = "Dodirni za potpis";`

I will compare it to the German so it is easier to unterstand. There is no big difference in addressing an older person and a friend in English:
e.g. 
to friend: "How are you?"
to older person: "How are You?"

but in German there is a difference:
to friend: "Wie geht es dir?"
to older person: "Wie geht es Ihnen?"

The same case is in Serbian and Croatian
to friend: "Kako si?"
to older person: "Kako ste?"

The first issue is not using of infinitive:
`"Dodirni za potpis"` would be in German `"Tipp zum Signieren"`
And the second issue is mixing of way of addressing someone (Croatian and Serbian are almost identical):
`"Dodirnite da biste potpisali"` would be in German `"Tippen Sie zum Signieren"`

I could create a Pull Request, but I need to know what convention do you use?
Most likely there is the same issue with other languages, but I noticed this in Serbian and Croatian.

**Comment by MarcoEidinger** [comment]
Dear @Kukurijek, I appreciate you are taking the time to provide feedback regarding the localized texts in this project, thank you! I'll have to check with our translation coordinator on how to proceed. Let me come back to you in a bit.

**Comment by MarcoEidinger** [comment]
Dear @Kukurijek, our translation coordinator will request to check all languages to use infinitive form and to make corrections if necessary. If you can propose translations for Serbian and Croatian (simply via commenting on this issue) then that would be helpful input :)

**Comment by Kukurijek** [comment]
Hi @MarcoEidinger 
"Tap to Sign" would be `"Dodirnuti za potpis"` in both languages (Serbian and Croatian), but this is just an example for one localized text.

I will try to fix all of them, at least in these two languages.

Thanks :)

**Comment by MarcoEidinger** [comment]
Dear @Kukurijek , our tranlsation coordinator needs more time to verify your proposal for compliance with the SAP Translation Style Guides. Use of imperative vs infinitive form seems to vary from language to language. We'll get back to you next week.

**Comment by MarcoEidinger** [comment]
Hi @Kukurijek,

I already left the comment in your pull request but it is important and therefore I repeat it:  I want to let you know that I very much appreciate your input! You took time off your busy day to provide feedback which is something special. You challenged the translation team to review the current translation. This is a good thing and on behalf of the team I say "Thank you" !!

Unfortunately, the given feedback cannot be accepted as a whole as the verbs in question should be left in the imperative form according to the SAP translation style guides. Hence I am closing this issue.

Have a great day

**Comment by Kukurijek** [comment]
Hi @MarcoEidinger,

Now I know what the rules of the  SAP translation style guides are 😄 

 Just one small nit pick on some translation style :) ->
 
 As I already mentioned, there is still a mix of informal and formal way of addressing.
 I have explained it on German example, because it is easier to understand, but there are almost same rules for Serbian and Croatian. I think, it should be reviewed again, maybe 😅 
 
Maybe I did not unterstand the translation style guides, but it does not make so much sense to mix the way of addressing on the different screens for instance.
 
 sr:
```
"Save" = "Sačuvaj"; --- translated to German would be "Speichere"

"Re-enter Signature" = "Ponovo unesite potpis";  ---  translated to German would be "Geben Sie die Signatur wieder ein"
```

I just want to say that I still do not see the logic behind this.

If you compare Croatian and Serbian, you will notice a mix again ->

sr:
`"Re-enter Signature" = "Ponovo unesite potpis";`

cr:
`"Re-enter Signature" = "Ponovo unesi potpis";`

There is no difference between Serbian and Croatian in this case, but the provided translations are different - again is way of addressing mixed.

And just want to support translators and get them on the right track 😄 

Thanks you for the kind answer!

Have a great day too🚀
